src/sentry/testutils/factories.py

```diff
@@ -1169,6 +1169,9 @@ def create_group(project, create_open_period=True, **kwargs):
         if "status" not in kwargs:
             kwargs["status"] = GroupStatus.UNRESOLVED
             kwargs["substatus"] = GroupSubStatus.NEW
+        if "status" in kwargs:
```

Comment:
This is an unrelated change but I always see an error when executing tests locally.